pytest_operator/plugin.py

```diff
@@ -65,17 +66,17 @@ def pytest_addoption(parser):
     )
     parser.addoption(
         "--crash-dump",
-        action="store_true",
+        action=argparse.BooleanOptionalAction,
```

Comment:
this only seems available in [Python3.9](https://docs.python.org/3/library/argparse.html#action)

Reply:
Thanks I will remove it.

Reply:
I replaced `--crash-dump` with `--no-crash-dump`. The `--no-crash-dump` disables `juju-crashdump` from running automatically after failed tests.